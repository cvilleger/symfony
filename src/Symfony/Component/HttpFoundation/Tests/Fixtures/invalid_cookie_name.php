<?php

use Symfony\Component\HttpFoundation\Cookie;
use Symfony\Component\HttpFoundation\Response;

require __DIR__ . '/common.inc';

$r = new Response();
try{
    $r->headers->setCookie(new Cookie('Hello + world', 'hodor'));
} catch (\InvalidArgumentException $e){
    return null;
}
$r->sendHeaders();
