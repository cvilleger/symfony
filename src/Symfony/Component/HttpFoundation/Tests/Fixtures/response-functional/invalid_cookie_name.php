<?php

use Symfony\Component\HttpFoundation\Cookie;
use Symfony\Component\HttpFoundation\Response;

$r = require __DIR__ . '/common.inc';

try{
    $r->headers->setCookie(new Cookie('Hello + world', 'hodor'));
} catch (\InvalidArgumentException $e){
    return null;
}
$r->sendHeaders();
