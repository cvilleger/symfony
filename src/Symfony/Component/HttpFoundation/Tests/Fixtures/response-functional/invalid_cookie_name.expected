
Array
(
    [0] => Content-Type: text/plain; charset=utf-8
)
shutdown
